src/Twig/Handler/ImageHandler.php

```diff
@@ -155,12 +155,20 @@ public function showImage($fileName = null, $width = null, $height = null, $crop
         }
         $thumb = $this->getThumbnail($fileName, $width, $height, $crop);
 
-        if ($width === null || $height === null) {
+        if ($width === null && $height === null) {
+            $thumbconf = $this->app['config']->get('general/thumbnails');
+            $width = !empty($thumbconf['default_image'][0]) ? $thumbconf['default_image'][0] : 1000;
+            $height = !empty($thumbconf['default_image'][1]) ? $thumbconf['default_image'][1] : 750;
+            $thumb->setWidth($width);
+            $thumb->setHeight($height);
```

Comment:
Yeah, but there it assumes that you want the default_thumbnail defaults, here we want the default_image defaults. This is basically the same thing that we already do in the `popup` function.